File: python/put_journal.py
from flask import Flask, jsonify, abort, make_response, request, render_template
import requests, json, datetime, time

app = Flask(__name__)
start = time.time();

# Request to API

# Make are roots
@app.route('/')
def hello_world():

    global start, time

    url = "http://localhost:2000/journals/edit/114"

    payload = "title=The%20Sun%20is%2C%20oh%2C%20so%20desperate%20to%20set%20tonight"
    headers = {
        'x-access-token': "52$QS^OVNjz4AnbWjwbyZ*I8F&MgIXXRDPm2XFDSJDpX8dUmP^zLVO#tE-g&yk)O",
        'cache-control': "no-cache",
        'postman-token': "a89c7582-3222-70b0-39af-ed2ba7f8ad92",
        'content-type': "application/x-www-form-urlencoded"
    }

    response = requests.request("PUT", url, data=payload, headers=headers)

    end    = time.time(); # microseconds
    output = round((end - start) / 1000, 4) # convert to milliseconds

    # start = 0.0
    end = 0.0
    # Output
    return render_template('journals.html', result=response.text, time = output)

File: php/put_journal.php
<?php

$time_start = microtime(true);

$curl = curl_init();

curl_setopt_array($curl, array(
  CURLOPT_PORT => "2000",
  CURLOPT_URL => "http://localhost:2000/journals/edit/112",
  CURLOPT_RETURNTRANSFER => true,
  CURLOPT_ENCODING => "",
  CURLOPT_MAXREDIRS => 10,
  CURLOPT_TIMEOUT => 30,
  CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
  CURLOPT_CUSTOMREQUEST => "PUT",
  CURLOPT_POSTFIELDS => "title=The%20Sun%20is%2C%20oh%2C%20so%20desperate%20to%20set%20tonight",
  CURLOPT_HTTPHEADER => array(
    "cache-control: no-cache",
    "content-type: application/x-www-form-urlencoded",
    "postman-token: 58b01fe4-086a-105c-77ec-20c1027e7dd5",
  ),
));

$response = curl_exec($curl);

curl_close($curl);

echo $response;

// $end = strtotime("now");
// $output = $start - $end / 1000;
$time_end = microtime(true);
$time = number_format($time_end - $time_start, 4);
echo "<hr>";
echo "\nProcess time execution: $time seconds\n";
